### PR TITLE
Update train-to-teach-in-england-as-an-international-student.md

### PR DESCRIPTION
**Pull request by laurahermionetennant92**
Work in progress

### Trello card
https://trello.com/c/yWQGKhHG/253-iterate-content-on-git-international-trainee-page

### Context
Updating content in light of Nexer usability research

### Changes proposed in this pull request

WIP – will review when preview is generated

### Guidance to review

No need to review yet :)

### CODE REVIEW

**Comment by gemmadallmandfe** [comment]
@laurahermionetennant92 I've converted this to Draft whilst you are still working on it. Let us know when it is ready to review!

**Comment by laurahermionetennant92** [comment]
@peteryates Morning, I made some more changes to this PR last night but there's no new preview – does it just take a while to come through? Thank you! :)

**Comment by laurahermionetennant92** [comment]
Hi @EllieNodder I think I'm finally read to publish this monster. Have you got capacity for 2i?
Just looking for typos at this stage please :)

File: app/views/content/train-to-teach-in-england-as-an-international-student.md [hunk 1]
```diff
@@ -58,66 +50,100 @@ keywords:
...
 
 Visit the [UK Council for International Student Affairs](https://www.ukcisa.org.uk/) for lots of helpful information about studying at an English university.
 
-## 2. Get help and advice
+## 2. Get support and advice
+
+To get advice about training to teach, you can:
 
-### Help with international qualifications
+* call 0800 389 2500 for free from anywhere in the world, Monday to Friday between 8.30am and 5.30pm, Greenwich Mean Time
```
Comment (peteryates) [inline app/views/content/train-to-teach-in-england-as-an-international-student.md:116]: Is this definitely always GMT or should it be UK local time? (i.e., does the helpline observe daylight savings?)

**Comment by laurahermionetennant92** [comment]
@valentine-carter I just removed a rogue full stop from the opening par. Happy for this to be merged once that's done :)

**Review by gemmadallmandfe — COMMENTED** [review]
@valentine-carter Some suggestions to consider

The comment re dates is the most important one - the dates are not right and there is still discussion around them

File: app/views/content/train-to-teach-in-england-as-an-international-student.md (hunk 1 above)
Comment (valentine-carter) [reply app/views/content/train-to-teach-in-england-as-an-international-student.md]: I agree. We've changed this.

File: app/views/content/train-to-teach-in-england-as-an-international-student.md [hunk 2]
```diff
@@ -58,69 +50,100 @@ keywords:
...
+* training lasts one 1 academic year full-time, or longer part-time
+* you can apply for paid teaching jobs as soon as you successfully complete your training
+* you'll earn at least £25,000 a year, and £32,000 in London, when you qualify as a teacher
+* you can update, extend or switch your visa to live and work in the UK long-term
+
+## Application timeline
+
+Some dates for your diary:
+
+* 21 September 2022: deadline to apply for courses starting in autumn 2022 (it's best to apply as soon as you can)
```
[the remark this answers is not in the document]
Comment (valentine-carter) [reply app/views/content/train-to-teach-in-england-as-an-international-student.md:90]: I'm going to delete this whole section for now.

File: app/views/content/train-to-teach-in-england-as-an-international-student.md [hunk 3]
```diff
@@ -58,69 +50,100 @@ keywords:
...
+* international candidates can expect to pay between £12,000 and £22,000, plus
+living costs, to train (depending on your immigration status, you may get a reduction in fees and some financial support)
+* training lasts one 1 academic year full-time, or longer part-time
+* you can apply for paid teaching jobs as soon as you successfully complete your training
+* you'll earn at least £25,000 a year, and £32,000 in London, when you qualify as a teacher
+* you can update, extend or switch your visa to live and work in the UK long-term
+
+## Application timeline
+
+Some dates for your diary:
```
[the remark this answers is not in the document]
Comment (valentine-carter) [reply app/views/content/train-to-teach-in-england-as-an-international-student.md:88]: It needs a lead-in line style-wise. It does look weird without to be fair.

File: app/views/content/train-to-teach-in-england-as-an-international-student.md [hunk 4]
```diff
@@ -58,69 +50,98 @@ keywords:
...
+- international candidates can expect to pay between £12,000 and £22,000, plus
+living costs, to train (depending on your immigration status, you may get a reduction in fees and some financial support)
+- training lasts one 1 academic year full-time, or longer part-time
+- you can apply for paid teaching jobs as soon as you successfully complete your training
+- you'll earn at least £25,000 a year, and £32,000 in London, when you qualify as a teacher
+- you can update, extend or switch your visa to live and work in the UK long-term
+
 
-## Key facts
+* 12 October 2022: applications open for courses starting in
```
Comment (EllieNodder) [inline app/views/content/train-to-teach-in-england-as-an-international-student.md:87]: think the end of this sentence has ran away

File: app/views/content/train-to-teach-in-england-as-an-international-student.md [hunk 5]
```diff
@@ -130,84 +151,90 @@ You can apply for a salaried teacher training course, when you’ll earn money w
...
 
-## 8. Apply for your first teaching job
+### 8. Apply for your first teaching job
 
 When you have qualified teacher status, you will be able to apply for paid teaching jobs in all schools in England.
 
-<a href="https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#job" class="button">Learn more about your first teaching job in England</a>
+<a href="https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#job">You can learn more about your first teaching job in England</a>
 
+<a href="https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#job">Learn more about your first teaching job in England</a>
```
Comment (EllieNodder) [inline app/views/content/train-to-teach-in-england-as-an-international-student.md:236]: Repeats above link. Assume this was to replace the above and the above should be deleted?

File: app/views/content/train-to-teach-in-england-as-an-international-student.md [hunk 6]
```diff
@@ -58,69 +50,98 @@ keywords:
...
+If you do not have qualifications that are equivalent to GCSEs, you may be able to sit a test set by training providers to show your knowledge of English, maths and science.
 
-To teach children aged 3 to 11 in primary school, you will also need a standard in physics, chemistry or biology equivalent to a grade 4 GCSE.
+[Contact us for advice about this](#get-support-and-advice)
 
-A GCSE is an academic qualification awarded for exams in England, usually taken at age 16. [Get help understanding English qualifications and their international equivalents](#get-help-and-advice).
+#### If you do not have a university degree
 
-### If you do not have a university degree
+If you don't have a degree, apply to study a 3 to 4 year [undergraduate course in England](https://www.ucas.com/postgraduate/teacher-training/applying-teacher-training/find-teacher-training-programmes) which combines a teaching degree with [qualified teacher status](https://www.gov.uk/guidance/qualified-teacher-status-qts) (QTS). When you complete your studies, you’ll be able to apply for jobs as a teacher in England.
```
Comment (EllieNodder) [inline app/views/content/train-to-teach-in-england-as-an-international-student.md:111]:
```suggestion
If you don't have a degree, apply to study a 3 to 4 year [undergraduate course in England](https://www.ucas.com/postgraduate/teacher-training/applying-teacher-training/find-teacher-training-programmes) which combines a teaching degree with [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts). When you complete your studies, you’ll be able to apply for jobs as a teacher in England.
```

File: app/views/content/train-to-teach-in-england-as-an-international-student.md [hunk 7]
```diff
@@ -130,84 +151,90 @@ You can apply for a salaried teacher training course, when you’ll earn money w
   </ul>
 </div>
 
-<a href="https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#rate" class="button">Check your eligibility for financial support</a>
+<a href="https://www.gov.uk/government/publications/train-to-teach-in-england-non-uk-applicants/train-to-teach-in-england-if-youre-a-non-uk-citizen#rate">Check your eligibility for financial support</a>
 
+### 4. Search and apply for teacher training courses
 
-## 4. Search and apply for teacher training courses
+You can [find postgraduate teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) in England leading to [qualified teacher status](https://www.gov.uk/guidance/qualified-teacher-status-qts) (QTS).
```
Comment (EllieNodder) [inline app/views/content/train-to-teach-in-england-as-an-international-student.md:158]:
```suggestion
You can [find postgraduate teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) in England leading to [qualified teacher status (QTS)](https://www.gov.uk/guidance/qualified-teacher-status-qts).
```

**Review by gemmadallmandfe — APPROVED** [review]

**Pull request merged**